README.md

```diff
@@ -1 +1,66 @@
-# tf_efs_backup
\ No newline at end of file
+# tf_efs_backup
+
+# Terraform module for automatic EFS backup
+
+This repo contains a terraform module that creates backup of EFS Filesystems
+
+The workflow is simple:
+* Periodically launch resource (EC 2 instance) based on schedule
+* Execute the shell command defined in the activity on the instance
+* Execute sync data from Production EFS to S3 Bucket by aws-cli
+* The execution log of the activity is stored in S3
+* Publish to the SNS topic that defined the success or failure of the activity
+* Automatic backup rotation using `S3 lifecycle rule`
+
+## Usage
+
+Include this repository as a module in your existing terraform code:
+
+Notes:
+
+```
+module "efs_backup" {
+  source = "git::https://github.com/cloudposse/tf_efs_backup.git?ref=master"
+
+  name                      = "${var.name}"
+  stage                     = "${var.stage}"
+  namespace                 = "${var.namespace}"
+  region                    = "${var.region}"
+  vpc_id                    = "${var.vpc_id}"
+  efs_ids                   = "${var.efs_ids}"
+  s3_bucket_expiration_days = "${var.s3_bucket_expiration_days}"
```

Comment:
Rename to `noncurrent_version_expiration_days` as it's more clear what it refers to